### ISSUE TITLE
Enable meteocool for x hours, disable after that time

### ISSUE THREAD

**Issue by philphilphil** [v4lli/meteocool#144]
Most of the time I'm Indoors and don't need to know when it rains.

I would like a functionality, where i can enable meteocool for a given amount of time. Eg. "Enable now for 4h". It then sends my location (maybe even more often than normally) and checks for rain. After the 4h location sending and receiving notifications will be disabled again.

It would save battery life and backend resources.

In my case it's the Android-App where i'd like that feature but i'm sure its also useful for iOS users.

**Comment by pew** [comment]
Seems like a feature you'd always forget to enable since it requires a manual step. IMHO it makes sense if the battery drain is huge.

**Comment by philphilphil** [comment]
On most days mostly the entire day I don't care if it rains. Only when I plan to go outside (or am outside) I need to know when and if it will rain and that's when i want the notifications. I won't forget to turn it on.
For users who will maybe some kind of scheduler would be helpful. Eg. "disable Mo-Fr from 0:00 till 17:00" (the time I'm at work and don't care if it rains)

**Comment by v4lli** [comment]
@philphilphil would a "Mute until the weather gets better again" button suit your needs as well? The backend could wait until there hasn't been any rain for e.g. 24h at the user's location before sending notifications again.

**Comment by philphilphil** [comment]
Yes that would work if the "timeout" is set much lower. Maybe after 1h of no rain it gets active again.
Additionally maybe just mute options to mute for 2h or 24h or something?
